css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,19 @@
+button {
+  font-size: 20px;
+  padding: 30px 30px 22px 22px;
+}
+
+button:hover {
+  color: white;
+  background-color: black;
+  cursor: url("hand.cur"), pointer;
+}
```

Comment:
> Whenever the user hovers the button, the cursor changes to a pointer hand. I looked at the the instructions and checked out the link to cursor. I used what the examples provided.

@chrisahn1  Oh, I see where you got it from, but `url("hand.cur")` will only work if your project contains a custom cursor image file named `hand.cur`. Since this exercise doesn't have one, and the instructions ask you to use the built-in `pointer` cursor, please leave that part out.